WordPress/twentytwentyone
# The password input field is the wrong size

Fixes #449

## Summary

Added a the_password_form filter 
## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Create a password protected post/page
1. Check the password input field on the frontend


## Screenshots

If this is a visual change please include screenshots of the change at various screen sizes.

<details>
<summary>Desktop (XL, L) + TABLET</summary>

![image](https://user-images.githubusercontent.com/20287474/95990565-f911ad80-0e2b-11eb-8f14-0474fd58bc36.png)


</details>

<details>
<summary>Mobile (Small)</summary>

![image](https://user-images.githubusercontent.com/20287474/95990613-0a5aba00-0e2c-11eb-9ff4-44206d0428a5.png)


</details>

## Quality assurance
